--- disk_usage_original.py	2021-05-25 23:19:06.000000000 +0100
+++ disk_usage_fixed.py	2021-05-25 23:19:33.000000000 +0100
@@ -1,5 +1,6 @@
 
 import shutil
+import sys
 
 
 def check_disk_usage(disk, min_absolute, min_percent):
@@ -19,9 +20,9 @@
 
 # Check for atleast 2 Gig and 10% free
 
-if not check_disk_usage('/', 2*2**30, 10):
+if not check_disk_usage('/', 2, 10):
     print("EROOR! Not enough disk space")
-    return 1
+    sys.exit(1)
 
 print("Everything is ok")
-return 0
+sys.exit(0)
